### PR TITLE
DistGitPRHandler stub

### PR DESCRIPTION
**Pull request by jpopelka**
For now, we build the hardly image from packit-worker image, we probably want to decouple this, but I'm not sure how.

Do we want the run of `DistGitPRHandler` to be configurable in config  (`dist_git_pr` job would need to be added to packit's `JobType` and to all projects` configs) or will it run (as it is for now) by default for any project?

### CODE REVIEW

File: hardly/tasks.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+) -> List[TaskResults]:
+    """
+    Base celery task for processing messages.
+
+    :param event: event data
+    :param topic: event topic
+    :param source: event source
+    :return: dictionary containing task results
+    """
+    return StreamJobs().process_message(event=event, topic=topic, source=source)
```
Comment (lachmanfrantisek) [inline hardly/tasks.py:60]: What about using a different name than `process_message` to not collide and be more explicit when calling this.

Or, is this for some reason?

**Review by lachmanfrantisek — COMMENTED** [review]
Looks good to me.

File: hardly/tasks.py (hunk 1 above)
Comment (jpopelka) [reply hardly/tasks.py]: Not sure I understand. We don't collide, we override it. But I renamed it (to `hardly_process`) if that's better for you.

Comment (lachmanfrantisek) [reply hardly/tasks.py]: I meant the `process_message` in `tasks.py` and the task name and was not sure if this is the same for purpose. (And as you wrote above, we do that to override it. That answered my question...;-)

**Review by lachmanfrantisek — APPROVED** [review]

**Pull request merged**